Trimble.Modus.Components/Controls/Toast/TMToastContents.xaml.cs

```diff
@@ -121,17 +121,17 @@ private void PopupData(string message, string actionButtonText, Action action, b
                     action?.Invoke();
                 };
 
-                if (ToastBackground.Equals(ResourcesDictionary.ColorsDictionary(ColorsConstant.BluePale)))
+                if (ToastBackground.Equals(ResourcesDictionary.ColorsDictionary(ColorsConstants.BluePale)))

```

Comment:
Can you change this code with switch case

Reply:
We cannot use `ResourcesDictionary.ColorsDictionary(ColorsConstants.BluePale)` in switch case